test/qtismtest/runtime/expressions/operators/PatternMatchProcessorTest.php

```diff
@@ -110,7 +110,8 @@ public function patternMatchProvider()
             [new QtiString('string'), 'shell', false],
             [new QtiString('stringString'), '.*', true], // in xml schema 2, dot matches white-spaces
             [new QtiString('^String$'), 'String', false], // No carret nor dollar in xml schema 2
-            [new QtiString('^String$'), '^String$', true],
+            // no aplicable because commit [ db4d9a49 ]: / fix: update removing caret and dollar, update tests /
+//            [new QtiString('^String$'), '^String$', true],
```

Comment:
This one is probably an invalid use case.
The pattern is correct, while the value indeed cannot be expected to be matched when it contains the caret and dollar signs, as they are treated as plain text.
Feel free to remove that use case.